### PR TITLE
task 5 is done

### PR DESCRIPTION
**Pull request by kellysum**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -95,6 +95,38 @@ describe("GET /api",()=>{
             })
             
         })
+describe("GET /api/articles", ()=>{
+    test('200: responds with an array of article object', ()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body})=>{
+            const {articles} = body
```
Comment (popatre) [inline __tests__/app.test.js:104]: Need to test for this:

the articles should be sorted by date in descending order.

You can use jest sorted for this.

File: model/articles.model.js [hunk 2]
```diff
@@ -12,3 +12,16 @@ exports.selectArticleByArticleId = (article_id)=>{
...
 
+exports.selectArticles = ()=>{
+    return db.query(`
+    SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, 
+    CAST (COUNT (comments.article_id) AS INT) AS comment_count 
+    FROM articles
+    LEFT JOIN comments ON comments.article_id = articles.article_id
+    GROUP BY articles.author, title, topic, articles.created_at, articles.votes, article_img_url, articles.article_id
+    ORDER BY created_at DESC;`
+    )
```
Comment (popatre) [inline model/articles.model.js:22]: Excellent join, well done!

**Review by popatre — CHANGES_REQUESTED** [review]
Some lovely code here:
Need to test for this:

the articles should be sorted by date in descending order.

You can use jest sorted for this.

**Review by nc-mick-fay — APPROVED** [review]
Hi Kelly, good addition of your sorting test - this is ready to merge into main when you're ready!

**Pull request merged**